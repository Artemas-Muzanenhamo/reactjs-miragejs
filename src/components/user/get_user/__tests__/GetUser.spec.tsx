import React from "react";
import renderer from "react-test-renderer";
import {GetUser} from "../GetUser";

describe('<GetUser /> component', () => {
    it('renders <GetUser /> component', () => {
        let userComponentAsJson = renderer.create(
            <GetUser name="Jane" surname="Doe"/>
        ).toJSON();

        expect(userComponentAsJson).toMatchSnapshot();
    });
});
