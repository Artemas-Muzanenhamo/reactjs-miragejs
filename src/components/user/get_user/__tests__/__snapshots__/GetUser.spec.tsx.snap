// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GetUser /> component renders <GetUser /> component 1`] = `
<div>
  <div
    className="uk-card uk-card-primary uk-card-body"
  >
    <h3
      className="uk-card-title"
    >
      Jane
       
      Doe
    </h3>
    <p>
      This is a profile card for 
      Jane
    </p>
  </div>
</div>
`;
